### PR TITLE
Reviewing and fixing 'Production Deployment Guidelines' doc

### PR DESCRIPTION
**Pull request by rmsamitha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md [hunk 1]
```diff
@@ -62,10 +65,13 @@ Prior to installing any WSO2 Carbon-based product, it is necessary to have the a
...
--   By default, WSO2 products are installed with **OpenJDK** , which allows you to run the product as soon as it is installed.
+-   If you are using the product installer to install the product, by default, WSO2 API-M is installed with OpenJDK, 
+which allows you to run the product as soon as it is installed. If you are using the product's binary distribution to
+ install the product (instead of the product installer), install JDK. Make sure your JDK version is compatible with 
+ the WSO2 product.
 
-    !!! tip
-        To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
+!!! tip
+    To use a different JDK, point the **`JAVA_HOME`** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
```
Comment (Meruja) [inline en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md:74]: shall we remove the extra space before the full stop?

Comment (rmsamitha) [reply en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md]: Fixed with dd3957d

File: en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ JMX is enabled in WSO2 products by default, which ensures that the JMX server st
 
 ### Configuring JMX ports for the server
 
-The default JMX ports (RMIRegistryPort and the RMIServerPort) are configured in the `carbon.xml` file (stored in the `<PRODUCT_HOME>/repository/conf` directory) as shown below. If required, you can update these default values.
+The default JMX ports (RMIRegistryPort and the RMIServerPort) can be changed by adding the below config into 
+`<PRODUCT_HOME>/repository/conf/deployment.toml`file as shown below. Below are the default values and you can update
```
Comment (menakaj) [inline en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md:12]: `<PRODUCT_HOME>/repository/conf/deployment.toml`file as shown below.

There should be a space before **'file'**

File: en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md [hunk 3]
```diff
@@ -8,24 +8,24 @@ JMX is enabled in WSO2 products by default, which ensures that the JMX server st
 
 ### Configuring JMX ports for the server
 
-The default JMX ports (RMIRegistryPort and the RMIServerPort) are configured in the `carbon.xml` file (stored in the `<PRODUCT_HOME>/repository/conf` directory) as shown below. If required, you can update these default values.
+The default JMX ports (RMIRegistryPort and the RMIServerPort) can be changed by adding the below config into
```
Comment (menakaj) [inline en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md:11]: below config => following config

File: en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md (hunk 2 above)
Comment (menakaj) [reply en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md:12]: Below are the default values and you can update these default values as required. => Following are the default values and you can update them as required

File: en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md (hunk 3 above)
Comment (rmsamitha) [reply en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md:11]: What is the difference between these two?

File: en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md (hunk 2 above)
Comment (rmsamitha) [reply en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md]: Fixed with 5086532aee5e568eda5989913fb757c3f564c6f3

**Pull request merged**